telemetry/metrics.go

```diff
@@ -409,6 +410,14 @@ var (
 	MetricConsensusDuration *prometheus.HistogramVec
 )
 
+var telemetryInitOnce sync.Once
+
+func init() {
+	telemetryInitOnce.Do(func() {
+		_ = SetHistogramBuckets("")
+	})
+}
+
```

Comment:
This is risky (in terms of ordering), where exactly is it needed? We can make run only in that flow. If it's for validation path we can run it there.

Reply:
yeah I agree. this is needed for the validation flow. will run it there